edx_rbac/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2020-03-16 06:24
+from __future__ import unicode_literals
+
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ConcreteUserRole',
```

Comment:
This migration shouldn't be in the `edx_rbac` app, since these models are only needed for tests.  You should treat the `tests` directory where these models are defined as a Django app, and put this migrations directory in it instead.  You should just need to add `tests` to `INSTALLED_APPS` in the test settings and define an [AppConfig](https://docs.djangoproject.com/en/2.2/ref/applications/#configuring-applications).

Reply:
We might need that because the current build is failing (after making the suggested changes) against Django 2.2, as `edx_rbac` is a Django app and it's referencing other Django models (with apps containing migrations), or we can just move models to `tests` app

Reply:
The only models in edx_rbac itself are abstract and have no associated tables.  The ones causing the tests to fail are the concrete subclasses of them in `tests/models.py`, so those are the ones running afoul of the rule you mentioned; that can be solved by turning `tests` into its own Django app.

Reply:
I quickly did a test to see if we really needed this migration by removing it. All tests pass even if this file doesn't exist.  I don't think it's necessary.  If you're still seeing an error when this file doesn't exist can you give me some steps to re-produce?